docs/chapters/ABOUT_ONEKO.md

```diff
@@ -0,0 +1,79 @@
+# About O-Neko
+
+O-Neko is a Kubernetes-native application that allows to cross a bridge between developers and other stakeholders by
```

Comment:
> allows to cross a bridge
sounds a bit strange to me. Maybe write
"that closes a gap between developers and other stakeholders"